app/views/restaurants/index.html.erb

```diff
@@ -1,3 +1,20 @@
 <% @restaurants.each do |restaurant| %>
-  <%= restaurant.inspect %>
+  <%= link_to restaurant do %>
+    <div class="col-sm-4 col-lg-4 col-md-4">
+      <div class="thumbnail transparency">
+        <div class="result-details">
+          <h4><%= restaurant.name %></h4>
+          <div>
```

Comment:
tnx.. changed it to address ( _header & index ) but it wouldn't change format though.. will take another look.